rfcs/text/0021-threat-enrichment.md

```diff
@@ -29,6 +24,22 @@ Stage 1: If the changes include field additions or modifications, please create
 Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting, and add them to the corresponding RFC folder.
 -->
 
+As these fields represent the enrichment of an existing event with indicator information, they are comprised of three categories of data:
+
+1. The indicator's indicator fields, as defined in RFC 0018
+2. Other relevant ECS fields from the indicator (see below)
+3. Fields representing the context of the enrichment itself
+
+### Proposed new fields
+
+Field | Type | Example | Description
+--- | --- | --- | ---
+matched.atomic | keyword | 2f5207f2add28b46267dc99bc5382480 | The value that matched between the event and the indicator
+matched.id | keyword | db8fb691ffdb4432a09ef171659c8993e6ddea1ea9b21381b93269d1bf2d0bc2 | The _id of the indicator document that matched the event
```

Comment:
I think that `matched.id` and `matched.index` could arguably just be `_id` and `_index` at the indicator level, but I don't think those are standard ECS fields, so looking for input on this here as well.

I think the argument for placing them within this `matched` fieldset is that the primary use is as metadata about the match (i.e. "what was the exact document that matched"), but these may make sense in a more general field 🤷

Reply:
Update: after some offline discussion and the knowledge that there exist no broad, general-purpose fields for elasticsearch-specific domain info like this, we're going to stick with `match.id` and `match.index` for now. When we fill out the corresponding Usage documentation about these fields, we'll want to note that their main purpose is auditing/debugging of enrichment logic.